pkg/metrics/metrics.jsx

```diff
@@ -445,13 +445,23 @@ class CurrentMetrics extends React.Component {
         }
 
         let cores = null;
+        let topCore = null;
         if (this.state.cpuCoresUsed.length > 1) {
             const top_cores = this.state.cpuCoresUsed.map((v, i) => [i, v]).sort((a, b) => b[1] - a[1])
                     .slice(0, 16);
             cores = top_cores.map(c => <Flex key={c[0]} justifyContent={{ default: 'justifyContentSpaceBetween' }}>
                 <FlexItem>{ cockpit.format(_("Core $0"), c[0]) }</FlexItem>
                 <FlexItem>{c[1]}%</FlexItem></Flex>
             );
+
+            topCore = <Progress
+                           id="current-top-cpu-usage"
+                           value={top_cores[0][1]}
+                           className="pf-m-sm"
+                           min={0} max={100}
+                           variant={ top_cores[0][1] > 90 ? ProgressVariant.danger : ProgressVariant.info }
+                           title={ _("Core with highest usage") }
```

Comment:
It should probably say "CPU", not "Core", as we call it CPU everywhere else. "Busiest CPU" sounded a bit strange to me. Bikeshedding appreciated!